plugins/modules/aci_interface_policy_leaf_policy_group.py

```diff
@@ -112,6 +113,66 @@
     - Choice of port_security_policy to be used as part of the leaf policy group to be created.
     type: str
     aliases: [ port_security_policy_name ]
+  link_flap_policy:
+    description:
+    - Choice of link_flap_policy to be used as part of the leaf policy group to be created.
```

Comment:
"to be created." same applies for to be created

Reply:
changed